src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -306,6 +322,16 @@ export class OmniboxResultList extends ResultList implements IComponentBindings
     }
   }
 
+  private handleQuerySuggestSelected(args: IQuerySuggestSelected) {
+    this.executeQueryHover(args.suggestion);
+  }
+
+  private executeQueryHover(suggestion: string) {
```

Comment:
I'm guessing this is incomplete ? As it does not actually seem like it execute anything meaningful, it just modifies an object without doing much with it.

Might be good to add some `TODO` in those case to help reviewers.